administration/src/bp-modules/cards/ImportCardsController.tsx

```diff
@@ -0,0 +1,101 @@
+import { NonIdealState, Spinner } from '@blueprintjs/core'
+import { ReactElement, useCallback, useContext } from 'react'
+import { useNavigate } from 'react-router-dom'
+
+import { WhoAmIContext } from '../../WhoAmIProvider'
+import CSVCard from '../../cards/CSVCard'
+import { Region } from '../../generated/graphql'
+import { ProjectConfigContext } from '../../project-configs/ProjectConfigContext'
+import useBlockNavigation from '../../util/useBlockNavigation'
+import StandaloneCenter from '../StandaloneCenter'
+import GenerationFinished from './CardsCreatedMessage'
+import CreateCardsButtonBar from './CreateCardsButtonBar'
+import ImportCardsInput, { FILE_SIZE_LIMIT_MEGA_BYTES } from './ImportCardsInput'
+import CardImportTable from './ImportCardsTable'
+import useCardGenerator, { CardActivationState } from './hooks/useCardGenerator'
+
+const FileRequirementsText = `Die Datei darf maximal ${FILE_SIZE_LIMIT_MEGA_BYTES} MB groß sein und muss im CSV Format sein.`
```

Comment:
i would prefer a small list with requirements like:
```js
- Die maximale Dateigröße beträgt: 2MB
- Dateiformat: CSV
- Spaltenformat: Name, Ablaufdatum, Kartentyp
- Gültiges Datumsformat: tt.mm.jjjj (Beispiel: 01.01.1970)
```

Reply:
I really like this idea. Will add this. Thank you :)